pkg/tc/filter_types.go

```diff
@@ -56,6 +63,26 @@ func ParseBasicParams(out io.Writer, params []string) (*tc.Object, error) {
 	return ret, nil
 }
 
+// Originally from tc filter u32 help
+// Usage: ... u32 [ match SELECTOR ... ] [ link HTID ] [ classid CLASSID ]
+//                [ action ACTION_SPEC ] [ offset OFFSET_SPEC ]
+//                [ ht HTID ] [ hashkey HASHKEY_SPEC ]
+//                [ sample SAMPLE ] [skip_hw | skip_sw]
+// or         u32 divisor DIVISOR
+
+// Where: SELECTOR := SAMPLE SAMPLE ...
+//        SAMPLE := { ip | ip6 | udp | tcp | icmp | u{32|16|8} | mark }
+//                  SAMPLE_ARGS [ divisor DIVISOR ]
+//        FILTERID := X:Y:Z
+
+// NOTE: CLASSID is parsed at hexadecimal input.
+
+const U32Help = `Usage ... u32 [ match ...] [ classid CLASSID | flowid CLASSID ]
+According to https://linux-tc-notes.sourceforge.net/tc/doc/cls_u32.txt
```

Comment:
Does that really belong in the help text, or is it a leftover comment?

nit: Imo something like `For further information see https://linux-tc-notes.sourceforge.net/tc/doc/cls_u32.txt` fits better

Reply:
I think, it is valuable to have the link in the help text to get a pointer to the complete filter syntax. 
 Changed to your wording.